client/assets/components/navbar.scss

```diff
@@ -0,0 +1,171 @@
+.top-navbar {
+  height: 70px;
+  border-radius: 0;
+  background-color: #fff;
+  margin: 0;
+  border: 0;
+  .navbar-brand {
+    height: 70px;
+    width: 70px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+}
+.navbar-collapse.collapse {
+  display: flex !important;
+  height: 70px !important;
+}
+.top-navbar__right-container {
```

Comment:
### Nitpick

Add a new line break here for readability sake. I've noticed a lot of your style definition are too nested, kindly add a line break to separate them so they're more readable.